### PR TITLE
fix: don't update label position when dragging labelled arrows

### PR DESCRIPTION
**Pull request by ad1992**
partially addresses https://github.com/excalidraw/excalidraw/issues/6882

### CODE REVIEW

**Comment by dwelle** [comment]
There's one case in which the label position is still being updated: when the arrow is inside a group and you move the group.

**Comment by ad1992** [comment]
> Checking, there's at least two more cases in which the label position is still being updated:
> 
> * [ ]  when the arrow is inside a group and you move the group
> * [x]  when the arrow is inside a frame and you move the frame

There is some issue when removing the position updates for labels in a group, could take some time to fix this.

**Comment by dwelle** [comment]
> There is some issue when removing the position updates for labels in a group, could take some time to fix this.

Could be solved by not adding the bound elements to groups. But first we'd have to align rendering and make the containerized text not rely on own position. https://github.com/excalidraw/excalidraw/issues/6883

**Comment by ad1992** [comment]
> > There is some issue when removing the position updates for labels in a group, could take some time to fix this.
> 
> Could be solved by not adding the bound elements to groups. But first we'd have to align rendering and make the containerized text not rely on own position. #6883

Yes I did try that but for labelled arrows its breaking which I will need to debug further why thats happening for labelled arrows even though we calculate position during render

**Comment by dwelle** [comment]
> Yes I did try that but for labelled arrows its breaking which I will need to debug further why thats happening for labelled arrows even though we calculate position during render

For me labelled arrows actually work if we don't include them in groups. What was the modality which didn't work?

**Comment by dwelle** [comment]
Let's merge this anyway as it's 2/3 of the way there. We'll fix the other cases later. (I'm keeping the issue open).

**Review by dwelle — APPROVED** [review]

File: src/tests/linearElementEditor.test.tsx [hunk 1]
```diff
@@ -1264,5 +1264,29 @@ describe("Test Linear Elements", () => {
         }),
       );
     });
+
+    it("should not update label position when arrow dragged", () => {
```
Comment (dwelle) [inline src/tests/linearElementEditor.test.tsx:1268]: (btw these tests are in the wrong file)

**Comment by ad1992** [comment]
> > Yes I did try that but for labelled arrows its breaking which I will need to debug further why thats happening for labelled arrows even though we calculate position during render
> 
> For me labelled arrows actually work if we don't include them in groups. What was the modality which didn't work?

When moving group including labelled arrows, it breaks for me. I am making the change in `dragSelectedElements.ts` and filtering the bound text elements from `elementsToUpdate`.
Probably you meant to exclude them specifically from `groups` instead

**Comment by dwelle** [comment]
> Probably you meant to exclude them specifically from `groups` instead

yup, basically so they have empty `groupIds`.

---
actually found another modality

- [ ] moving elements by resizing a frame (e.g. top-left frame resize handle) updates arrow label

**Pull request merged**